pkg/maps/lbmap/lbmap.go

```diff
@@ -398,6 +399,13 @@ func (*LBBPFMap) UpdateSourceRanges(revNATID uint16, prevSourceRanges []*cidr.CI
 
 	srcRangeMap := map[string]*cidr.CIDR{}
 	for _, cidr := range sourceRanges {
+		// k8s api server does not catch the IP family mismatch, so we need to catch it here
+		if ip.IsIPv6(cidr.IP) && !ipv6 {
+			log.WithFields(logrus.Fields{
+				logfields.ServiceID: revNATID,
+			}).Error("Source range's IP family does not match with the LB's")
```

Comment:
I'd use `Warn` instead of `Error`. Also, I'd log the `cidr` and the service frontend to make user's life easier.

Reply:
Added cidr in the log.  Service frontend is not available in the context. Cidr + the warning msg should be sufficient?